assets/src/scss/components/_buttons.scss

```diff
@@ -240,3 +240,7 @@ button.load-more-mt {
     }
   }
 }
+
+.post-content .wp-block-buttons {
```

Comment:
I think it would make more sense to add this to the [article-content style file](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/pages/post/_article-content.scss) instead of here, we already have the `.post-content` selector there!

Reply:
It can then be removed from here

Reply:
😅 I was trying to fix the e2e tests failing I forgot 🤦

Reply:
The tests are passing now!